### PR TITLE
Include 'cirq.' in various __repr__ outputs.

### PR DESCRIPTION
**Pull request by dlyongemallo**
Easy parts of #827

### CODE REVIEW

File: cirq/study/sweeps.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def param_tuples(self) -> Iterator[Params]:
         yield ()
 
     def __repr__(self):
-        return 'Unit'
+        return 'cirq.Unit'
```
Comment (Strilanc) [inline cirq/study/sweeps.py]: Hm, this one was wrong before. Should be `cirq.UnitSweep`

File: cirq/circuits/qasm_output.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def known_qasm_output(self,
                 self.theta, self.phi, self.lmda, qubits[0])
 
     def __repr__(self) -> str:
-        return 'QasmUGate({}, {}, {})'.format(self.lmda, self.theta, self.phi)
+        return 'cirq.QasmUGate({}, {}, {})'.format(self.lmda, self.theta, self.phi)
```
Comment (dabacon) [inline cirq/circuits/qasm_output.py]: line longer than 80 chars

File: cirq/study/sweeps.py [hunk 3]
```diff
@@ -176,7 +176,7 @@ def _gen(factors):
         return _gen(self.factors)
 
     def __repr__(self):
-        return 'Product({})'.format(', '.join(repr(f) for f in self.factors))
+        return 'cirq.Product({})'.format(', '.join(repr(f) for f in self.factors))
```
Comment (dabacon) [inline cirq/study/sweeps.py]: line longer than 80 char

File: cirq/study/sweeps.py (hunk 1 above)
Comment (dlyongemallo) [reply cirq/study/sweeps.py]: Done.

File: cirq/study/sweeps.py (hunk 3 above)
Comment (dlyongemallo) [reply cirq/study/sweeps.py]: Done.

File: cirq/circuits/qasm_output.py (hunk 2 above)
Comment (dlyongemallo) [reply cirq/circuits/qasm_output.py]: Done.

**Comment by dabacon** [comment]
No need to squash them in your fork.  We end up squashing on merge to master, so master is nice and beautiful.

**Pull request merged**